docs/validation.md

```diff
@@ -0,0 +1,26 @@
+VALIDATION:
+The validation is different for each database route; work was done using middleware.
+This is an example of how to implement it for the 'user-routes.ts' route
+
+
+--We import the following where authentication is to be implemented
+Configuration implemented in 'user-routes.ts'.
+
+import {schemaValidationMiddleware} from '../../middlewares/schemaValidator.middleware';
+import {postSchema, putSchema} from '../../schemmas/users.schema'
+
+--These instances are for POST and PUT; the following syntax should be used to apply them.
+
+    userRoutes.post('/users', schemaValidationMiddleware(postSchema), async (req, res) => {
```

Comment:
Please chance the name of the postSchema this validation is related with the user Wrapper model therefore the name should be reflect that